packages/core/src/modules/credentials/protocol/v2/CredentialMessageBuilder.ts

```diff
@@ -56,8 +56,19 @@ export class CredentialMessageBuilder {
     const filterAttachments: Attachment[] = []
     let credentialPreview: V2CredentialPreview | undefined
 
+    const props: CredentialExchangeRecordProps = {
+      connectionId: proposal.connectionId,
+      state: CredentialState.ProposalSent,
+      autoAcceptCredential: proposal?.autoAcceptCredential,
+      protocolVersion: CredentialProtocolVersion.V2,
+      credentials: [],
+    }
+
+    // Create the v2 record
+    const credentialRecord = new CredentialExchangeRecord(props)
```

Comment:
any reason over doing this? Saves an assignment, making the overall code more readable IMO

```suggestion
    // Create the v2 record
    const credentialRecord = new CredentialExchangeRecord({
      connectionId: proposal.connectionId,
      state: CredentialState.ProposalSent,
      autoAcceptCredential: proposal?.autoAcceptCredential,
      protocolVersion: CredentialProtocolVersion.V2,
      credentials: [],
    })
```

Reply:
I always prefer to create a const for the props; just a personal preference. But I will make the change, not a big deal.

Reply:
In that case leave it be, it's one personal preference over the other